File: backend/src/main/java/com/pequenospasos/backend/entity/Padre.java
package com.pequenospasos.backend.entity;

import com.pequenospasos.backend.enums.Role;
import jakarta.persistence.DiscriminatorValue;
import jakarta.persistence.Entity;

@Entity
public class Padre extends Usuario {


    public Padre() {
        super();
        this.setTipoUsuario(Role.PADRE);
    }

    public Padre(String nombre, String apellidos, String email, String password, String telefono) {
        super(nombre, apellidos, email, password, telefono, Role.PADRE);
    }
}

File: mobile-app/pequenosPasos/app/src/main/java/com/example/pequenospasos/data/model/Padre.kt
package com.example.pequenospasos.data.model

data class Padre(
    val id: Long,
    val nombre: String = "",
    val apellidos: String = "",
    val email: String = "",
    val password: String = "",
    val telefono: String = "",
    val tipoUsuario: String = "PADRE",
    val hijos: List<Nino> = emptyList()
)